src/beast/beast/nudb/store.h

```diff
@@ -861,14 +862,11 @@ store<Hasher, Codec, File>::commit()
         w.flush();
         s_->lf.sync();
     }
-    // VFALCO Audit for concurrency
-    {
-        std::lock_guard<gentex> g (g_);
-        // Write new buckets to key file
-        for (auto const e : s_->c1)
-            e.second.write (s_->kf,
-                (e.first + 1) * s_->kh.block_size);
-    }
+    g_.finish();
+    // Write new buckets to key file
+    for (auto const e : s_->c1)
```

Comment:
braces around this multiline for statement please.  

Or you could just make the second statement into one line, it gains nothing from being on two lines.